src/main/java/weizberg/gameoflife/GridComponent.java

```diff
@@ -105,4 +107,28 @@ public void clearButtonMethod() {
         }
         repaint();
     }
+
+    public void optionsButton(String option) throws IOException {
+        Path filePath = null;
+        switch (option) {
+            case "Glider":
+                filePath = Path.of(
+                        "C:\\Users\\weizb\\IdeaProjects\\Weizberg-Game-of-Life-2024\\src\\main\\java\\weizberg\\gameoflife\\files\\glider");
```

Comment:
Never, ever, ever put a path into git that only works on your machine.

Find a better way to do this.